distribution.md

```diff
@@ -104,3 +105,43 @@ The following labels MUST be set in the buildpack image(through the image config
 The buildpack ID and version MUST match a buildpack provided by a layer blob.
 
 For a buildpackage to be valid, each `buildpack.toml` describing a buildpack implementation MUST have all listed targets.
+
+### Run Image
+
+The following defines how a run image SHOULD be packaged for distribution as an OCI Image. The run image is the component that provides the base from which app images are built.
+
+The image configuration refers to the OCI Image configuration as mentioned [here](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+
+#### Image Configuration
+
+The Run Image MUST contain the following configurations:
+
+* Image Config's `config.User` field MUST be set to a non-root user with a writable home directory.
```

Comment:
Do we want to say that the user SHOULD be different from the build image's user?